tests/virt_autotest/login_console.pm

```diff
@@ -35,9 +35,12 @@ sub login_to_console() {
     assert_screen([qw(grub2 grub1)], 120);
     if (!get_var("reboot_for_upgrade_step")) {
         if (get_var("XEN") || check_var("HOST_HYPERVISOR", "xen")) {
-            send_key_until_needlematch("virttest-bootmenu-xen-kernel", 'down', 10, 1);
-            send_key 'ret';
+            if (!match_has_tag("virttest-bootmenu-xen-kernel")) {
+                send_key 'down';
```

Comment:
why send_key 'down' when below is send_key_until_needlematch pressing 'down' anyway

When I look on this whole change in login_console.pm I think it is not needed, maybe I'm wrong,
but the code before will do the same as the new code (pressing down until virttest-bootmenu-xen-kernel is found) or do I miss something ?